helix-rest/src/main/java/org/apache/helix/rest/server/service/VirtualTopologyGroupService.java

```diff
@@ -89,11 +89,6 @@ public VirtualTopologyGroupService(HelixAdmin helixAdmin, ClusterService cluster
    */
   public void addVirtualTopologyGroup(String clusterName, Map<String, String> customFields) {
     // validation
-    CloudConfig cloudConfig = _configAccessor.getCloudConfig(clusterName);
```

Comment:
What if someone wants to use it in Cloud environment. Can we just allow the config to be NULL and still proceed???

Reply:
It's totally fine to run in cloud environment. It doesn't matter whether its null or not. CloudConfig isn't used anywhere else after the validation.

Reply:
So it is like feature-flag kind of behavior.

Reply:
Not really, virtual group API was first only supposed to run in cloud environment, so `cloudConfig` was a prerequisite, now it's removed. There is no information or dependency needed from cloudConfig. A closer thing to feature-flag is `clusterConfig.isTopologyAwareEnabled`.

Reply:
thanks for explaining.